pymeasure/instruments/siglenttechnologies/siglent_spd1168x.py

```diff
@@ -38,21 +40,21 @@ def __init__(self, adapter, **kwargs):
         self.ch = {}
         self.ch[1] = SPDChannel(self, 1)
 
+    enable_4W_mode = Instrument.setting(
+        "MODE:SET %s",
+        """Configure 4-wire mode.
+
+        :type: bool
+            ``True``: enables 4-wire mode
+            ``False``: disables it.
+        """,
+        validator=strict_discrete_set,
+        values={False: "2W", True: "4W"},
+        map_values=True
+    )
+
     def shutdown(self):
         """ Ensure that the voltage is turned to zero
         and disable the output. """
-        self.ch[1].set_voltage(0.0)
-        self.ch[1].disable()
-
-
-if __name__ == "__main__":
-    psu = SPD1168X("TCPIP0::10.20.2.245::5025::SOCKET")
-    print(psu.id)
-    print(psu.selected_channel)
-    psu.ch[1].set_current = 1
-    psu.ch[1].disable()
-
-    print(psu.ch[1].set_voltage)
-    print(psu.ch[1].set_current)
-    print(psu.ch[1].voltage)
-    print(psu.error)
+        self.ch[1].voltage_setpoint = 0
+        self.ch[1].enable_output(False)
```

Comment:
also add a call to `super().shutdown()` at the end, so you get the rest of the cleanup, too.

Reply:
Also, since you seem to duplicate this in the other child class, maybe that should move up to `SPCBase`?